README.md

```diff
@@ -98,6 +98,7 @@ Read about guest lectures [here](course/guest-lectures.md).
 * [Hsiao-Ching (Jean) Lin](people/hsiaoching-lin.md)
 * [Huai-Che (Acsa) Lu](people/huaiche-lu.md)
 * [Inna Shteinbuk](people/inna-shteinbuk.md)
+* [Jacqueline J.Pang](people/jacqueline-pang.md)
```

Comment:
Hi Dan,
Thanks for reminds. Just correct it.
Is this correct from your side now?


On Wed, Sep 2, 2015 at 3:05 PM, Daniel Doubrovkine (dB.) @dblockdotorg <
notifications@github.com> wrote:

> In README.md
> <https://github.com/Cornell-CS5356-Fall2015/cs5356/pull/114#discussion_r38571184>
> :
>
> > @@ -98,6 +98,7 @@ Read about guest lectures [here](course/guest-lectures.md).
> >  * [Hsiao-Ching (Jean) Lin](people/hsiaoching-lin.md)
> >  * [Huai-Che (Acsa) Lu](people/huaiche-lu.md)
> >  * [Inna Shteinbuk](people/inna-shteinbuk.md)
> > +* [Jacqueline J.Pang](people/jacqueline-pang.md)
>
> Can I nitpick? There's a space after J. in the file :) It's ok, I'll let
> it slide.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cornell-CS5356-Fall2015/cs5356/pull/114/files#r38571184>
> .
>



-- 

Best Regards,
Jacqueline Pang
Candidate of Connective Media, MS.Information System, Cornell Tech, Cornell
University